protostar-rust/tests/data/declare_test/tests/test_declare.cairo

```diff
@@ -0,0 +1,8 @@
+use result::ResultTrait;
+use protostar_print::PrintTrait;
+
+#[test]
```

Comment:
Do we have an issue to extend those tests as requested?

Reply:
https://github.com/software-mansion/protostar/issues/2083